SU-HSDO/stanford_mrc
# MRCD8-181 MRCD8-187 MRCD8-198 MRCD8-224 Term & Node changes

# READY FOR REVIEW

# Summary
- Changes to node types: news, events, and visitor
- Applied patterns to views for visitor term pages.
- Changes from feedback no [this spreadsheet](https://docs.google.com/spreadsheets/d/1FFRXtxGtdyE4KmiAKBgumAu7gOw8E06BNcMnExr_pS8/edit#gid=542837041)

# Needed By (Date)
- 1/23

# Urgency
- High

# Steps to Test

1. Checkout branch.
2. `drush updb -y`
3. verify changes form the spreadsheet are reflected on the node edit forms.
4. create event series terms and content tagging those terms
5. view the term page.
6. ensure the content looks kinda like the [mock ups](https://projects.invisionapp.com/share/DCEGPUZS4#/screens/265034019)
7. verify the pathauto settings applied and the term lives at `events/event-series/[term-name]`
8. verify the "Official Name" is used as the page title.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
